README.md

```diff
@@ -222,6 +243,17 @@ Apprentices should memorize [**this**](/javascript/common-functions-cheatsheet.m
 1. [Negotiating](/career/interviewing/negotiating.md)
 1. [Ethics in Software Engineering](/apprenticeship-prep/ethics.md)
 
+### Weeks 15+ Final Project
+
+After Week 14, Participants build a full-stack web application in the course of one month, from inception to deployed to production. This includes:
```

Comment:
is there any documentation we could include here for people using this curriculum on their own? Don't we have project requirements in this repo somewhere?

Reply:
Will add!